app/src/main/java/com/pwhs/quickmem/presentation/app/home/recent/folder/AllRecentAccessFoldersUiState.kt

```
@@ -0,0 +1,4 @@
+package com.pwhs.quickmem.presentation.app.home.recent.folder
+
+class AllRecentAccessFoldersUiState {
+}
```

detekt (1.23.8)

[warning] The class or object AllRecentAccessFoldersUiState is empty.

(detekt.empty-blocks.EmptyClassBlock)